### ISSUE TITLE
Bug: when "enable screening" is on, it doesn't matter if the others are activated or not

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [joshfriend/GoFCCYourself#2]
The "missed call notification" and "call log" doesn't matter.
I don't see either of these.

See attached video:

[fLVrFx8Zgg.zip](https://github.com/joshfriend/GoFCCYourself/files/5140918/fLVrFx8Zgg.zip)

Also, BTW, had to do a lot to be able to import this project. Please make it easier... :(

[GoFCCYourself.zip](https://github.com/joshfriend/GoFCCYourself/files/5298771/GoFCCYourself.zip)

**Comment by joshfriend** [comment]
Yeah google released the API broken 3 years ago and it is still broken today:

* https://issuetracker.google.com/issues/151859054
* https://issuetracker.google.com/issues/138668201
* https://issuetracker.google.com/issues/130081372
* https://issuetracker.google.com/issues/166658116
* https://issuetracker.google.com/issues/221080455

**Comment by AndroidDeveloperLB** [comment]
@joshfriend Haha the first two of these are my reports.
On which Android versions did it work? And did it work fine on most/all devices on these versions?

**Comment by joshfriend** [comment]
Ha! thanks for continuing to file that bug after it keeps getting closed!

The call blocking aspect of this app works on all android versions since Q. I have only ever tried it on a Pixel 3XL and a Pixel 6. As we've both noted on the bugtracker though, the `setSkipNotification` and `setSkipCallLog` options do not work and notifications are always skipped and call logs are always appended. I recently added a comment to https://issuetracker.google.com/issues/221080455#comment12= noting that the Android 13 beta is still broken in this way.

**Comment by AndroidDeveloperLB** [comment]
I meant:
When were the other options working, that I've reported the bug about?

**Comment by joshfriend** [comment]
I'm not sure what you mean, sorry, setSkipNotification and setSkipCallLog are the only options I'm aware of besides blocking the call or allowing it through. Those are the two options that it looks like you filed bugs about:

* https://issuetracker.google.com/issues/166658116
* https://issuetracker.google.com/issues/221080455

Both of those reports are still not fixed.

**Comment by AndroidDeveloperLB** [comment]
Those are the options of the sample (which call setSkipNotification and setSkipCallLog) I'm talking about and that I've reported about.
I asked:
When (on which Android versions) have they ever worked?

**Comment by joshfriend** [comment]
They have _never_ worked as far as I can tell

**Comment by AndroidDeveloperLB** [comment]
@joshfriend So the sample shows something that never worked?
Too bad. 
As I wrote here, though, it seems that the function/s might work on some devices:
https://issuetracker.google.com/issues/221080455#comment4

**Comment by joshfriend** [comment]
OK! I see now...

I have not tested this app with anything but my Pixel 3XL and Pixel 6. `setSkipNotification` and `setSkipCallLog` have not worked to change the behavior on either phone on any version of android that I have tested on since the API was added.

The behavior of call logging might be reversed on my Pixel 6 though. It always appends the call log and I cannot skip adding blocked calls to the log.
